sanic_jwt/authentication.py

```diff
@@ -390,6 +388,8 @@ async def _verify(
         if token:
             try:
                 payload = await self._decode(token, verify=verify)
+                if self._extra_verifications:
+                    await self._verify_extras(payload, request)
```

Comment:
@ahopkins i have moved extra verifications here, to not pass explicit Request to all methods inside.
main reason - is to be able to access application context on verifications, for ex. to make calls to DB from sqlalchemy session inside app.ctx....

Reply:
You need to also check for `verify` otherwise it is a breaking change.

Reply:
done
btw, i thought to make `_extra_verifications` default to empty list like `_custom_claims`, to remove extra checks like `if _extra_verifications` to make code look cleaner... but not sure to place it in this MR or to make another one after